charts/osm/values.yaml

```diff
@@ -65,8 +65,8 @@ osm:
         # -- Average target CPU utilization (%)
         targetAverageUtilization: 80
       memory:
-        # -- Average target memory utilization (%)
-        targetAverageUtilization: 80
+        # -- Average target memory value
+        targetAverageValue: "50M"
```

Comment:
> memory values were set based on memory usage of the bookstore demo for osm-controller and osm-injector

@shalier the bookstore demo should not be used as a benchmark env to set default values. We need to use a more realistic testbed, such as one with 100 pods, so we can have defaults that work in most environments. In other words, using defaults based on the demo env is not something I agree with.